variables.tf

```diff
@@ -111,22 +111,22 @@ variable "load_balancer_sku" {
 }
 
 variable "managed_outbound_ip_count" {
-  default = null
+  default = 1
   type    = number
 }
 
 variable "outbound_ip_address_ids" {
   default = null
```

Comment:
Should an empty list be the default?

Reply:
No this is a number / count.

Reply:
No, next line defines it as list(any)...